backend/peoples/serializers.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework import serializers
+
+from peoples.models import Member
+
+
+class MemberDetailSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Member
+        fields = [
+            "name",
+            "mobile_number",
+            "nid_number",
+            "guardian_name",
+            "gender",
+            "serial_number",
+            "uuid",
+            "team",
+            "branch",
```

Comment:
branch value should be logged in staff's branch. No need to provide value from frontend.